src/jit/morph.cpp

```diff
@@ -6217,7 +6217,7 @@ bool                Compiler::fgCanFastTailCall(GenTreeCall* callee)
             // Get the size of the struct and see if it is register passable.
             if (argx->OperGet() == GT_OBJ)
             {
-#ifdef _TARGET_AMD64_
+#if defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_)
```

Comment:
See x64 unix logic that applies to arm64 as well.  The callee method could have a 16-byte struct param that should be counted as two 8-byte slots.